CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+ * New cops `AssertMatch` and `RefuteMatch` check for use of `assert_match`/`refute_match` instead of `assert(foo.match(bar))`/`refute(foo.match(bar))`
```

Comment:
Could you apply the following changelog entry format?
https://github.com/rubocop-hq/rubocop-minitest/blob/v0.5.1/CONTRIBUTING.md#changelog-entry-format